lib/utilities/pristine.js

```diff
@@ -178,106 +200,100 @@ function copyUnderTestApp(pristineAppPath, underTestAppPath) {
 }
 
 function addEmberToBowerJSON(appName, version) {
```

Comment:
That function is only called when `usesBower` is `true`. But then in a bower-less environment, we loose the capability to specify the version of Ember, using the `emberVersion` option, right?

Not sure how/if that worked before, we always added ember to bower, but not always removed ember-source. Does ember from bower take precedence of ember-source if both are present?

Anyways, to preserve the `emberVersion` feature in a bower-less world, we would have to add a `updateEmberSource(emberVersion)` like function to our setup chain, right?